doc/topics/windows/windows-package-manager.rst

```diff
@@ -817,30 +818,25 @@ Jinja:
         uninstall_flags: '/S'
       {% endfor %}
 
-In this example we are able to generate a software definition file that defines
-how to install 12 versions of Firefox. We use Jinja to create a list of
-available versions. That list is in a ``for loop`` where each version is placed
+In this example JINJA is used to to generate a package definition file that defines
+how to install 12 versions of Firefox. Jinja is used to create a list of
+available versions. The list is iterated through a  ``for loop`` where each version is placed
 in the ``version`` variable. The version is inserted everywhere there is a
 ``{{ version }}`` marker inside the ``for loop``.
 
-You'll notice that there is a single variable (``lang``) defined at the top of
-the software definition. Because these files are going through the Salt renderer
-many Salt modules are exposed via the ``salt`` keyword. In this case it is
-calling the ``config.get`` function to get a language setting that can be placed
-in the minion config. If it is not there, it defaults to ``en-US``.
+The single variable (``lang``) defined at the top of the package definition identifies the language of the package.
+You can access the Salt modules using the ``salt`` keyword. In this case, the ``config.get`` function is invokedto retrieve
```

Comment:
**Nitpick (grammar, blocking):** Insert a space in "invokedto."

Reply:
Updated